src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int base1;
+    private int base2;
+    private int height;
+
+    public IsoscelesTrapezoid(int base1, int base2, int height, String color) {
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+        this.setColor(color);
```

Comment:
If constructor that sets color is added in Figure class you could user instead of this.setColor(color) super(color).
super() refers to parent class object - in this case constructor from Figure that sets the color